src/sidebar/components/Annotation/test/AnnotationHeader-test.js

```diff
@@ -314,6 +330,23 @@ describe('AnnotationHeader', () => {
       });
     });
 
+    describe('Annotation share info', () => {
+      it('should render annotation share/group information if group is available', () => {
+        fakeIsHighlight.returns(false);
```

Comment:
Why does the highlight status matter here? Same query for the next test. Both tests pass if I comment the `fakeIsHighlight` lines out.

Reply:
Huh. For sure pilot error here. My best guess is that I copied some nearby test to use as "templates" and overlooked this. Removing.